src/components/organisms/Carousel.vue

```diff
@@ -1,60 +1,58 @@
 <template>
 	<div class="carousel">
-		<CarouselElement :imageContainer="imageContainer" :key="imageContainer.id" v-for="imageContainer in imageContainers"/>
-		<CarouselNavigation v-on:traverse="switchImage" :arrows="arrowContainer"/>
+		<CarouselElement :imageContainer="imageContainer" :key="index"  v-for="(imageContainer, index) in carousel.imageContainers"/>
+		<CarouselNavigation v-on:traverse="switchImage" :arrows="carousel.arrowContainer"/>
```

Comment:
if I have specified component for carousel arrows why I should pass the arrows to it? didnt you have arrows in atoms that can be used by defferent components